geniza/corpus/templates/admin/corpus/document/merge.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'admin/base_site.html' %}
+{% block title %} Merge selected documents {% endblock %}
+
+{% block breadcrumbs %}
+    {% if not is_popup %}
+        <ul>
+            <li><a href="{% url 'admin:index' %}">Home</a></li>
+            <li><a href="{% url 'admin:app_list' app_label='corpus' %}">Corpus</a></li>
+            <li><a href="{% url 'admin:corpus_document_changelist'%}">Document</a></li>
+            <li>Merge selected documents</li>
+        </ul>
+    {% endif %}
+{% endblock %}
+
+{% block content_title %}
+    <h1>Merge selected documents</h1>
+{% endblock %}
+
+{% block content %}
+    <form method="post">
+        {% csrf_token %}
+        {{ form.as_p }}
```

Comment:
Please add some light override css to adjust the form display to make it easier to read; you can probably just add these to our existing local admin css. 

I would like to see the bullets for the list suppressed and the radio input floated to the left of the document summary block.